BSSw blog post by Julie (#51)
Openscapes/website-new: blog/2023-08-17-bssw-fellowship/index.qmd

---
title: "Notes from my Better Scientific Software (BSSw) Fellowship"
# description: |
#  reflections of a first-time freelancer
author:
  - name: Julie Lowndes
    orcid: 0000-0003-1682-3872
date: 2023-08-17
citation: 
  url: https://openscapes.org/blog/2023-08-17-bssw-fellowship/
categories: [how-we-work]
image: openscapes-champions-series.png
---

*From Spring 2022-2023 I have had the pleasure of being a Better Scientific Software (BSSw) Fellow, by the Exascale Computing Project (17-SC-20-SC), a collaborative effort of the U.S. Department of Energy (DOE) Office of Science and the National Nuclear Security Administration; and by the National Science Foundation (NSF) under Grant No. 2154495. This is an excerpt from my final progress report to the The Krell Institute, who administered this award.*

*Quick links:*

-   [*How Open Source Tooling Is Changing the Way Professional Researchers Learn to Code*](https://bssw.io/blog_posts/how-open-source-tooling-is-changing-the-way-professional-researchers-learn-to-code) *- my post on the BSSw blog*

------------------------------------------------------------------------

> My focus during the fellowship was to support environmental science researchers through open source software that we share on GitHub. I modernized Openscapes' open educational resources that I first built with RMarkdown by refactoring them in Quarto, and I have since used these updated resources to teach 6 cohorts (50 teams) of professional environmental and Earth scientists at NASA, NOAA, and the EPA.

Reviewer: ```suggestion
My focus during the fellowship was to support environmental science researchers through open source software that we share on GitHub. I modernized Openscapes' open educational resources that I first built with RMarkdown by refactoring them in Quarto, and we have since used these updated resources to teach 6 cohorts (50 teams) of professional environmental and Earth scientists at NASA, NOAA, and the EPA.
```

> My focus during the fellowship was to support environmental science researchers through open source software that we share on GitHub. I modernized Openscapes' open educational resources that I first built with RMarkdown by refactoring them in Quarto, and I have since used these updated resources to teach 6 cohorts (50 teams) of professional environmental and Earth scientists at NASA, NOAA, and the EPA.

Reviewer: ```suggestion
My focus during the fellowship was to support environmental science researchers through open source software that we share on GitHub. I modernized Openscapes' open educational resources that I first built with RMarkdown by refactoring them in Quarto, and I have since used these updated resources to teach 6 cohorts (50 teams) of professional environmental and Earth scientists including NASA, and NOAA.
```

knitr::include_graphics("openscapes-champions-series.png")
```

::: {.small-text .center-text .dark-gray-text}
*A screenshot of the [Openscapes Champions Lesson Series](https://openscapes.github.io/series/) website.*
:::

I shared about this work with the BSSw community in two main ways. First, through a presentation for the [Exascale Computing Project HPC webinar](https://www.exascaleproject.org/event/openscapes/) ([slides](https://docs.google.com/presentation/d/1uKtVnZR8X94tADAWEE1THHKCbvfBbg2PBcH8asaoJQQ/edit#slide=id.g185866dc5bf_0_0)). This talk had three parts:

1.  Movement building: share Openscapes stories of open science for climate;

2.  Skill building: Quarto live demo;

3.  Onward: how you can help your colleagues and join the movement.

> This talk also explicitly addressed: why is this relevant to the Exascale community? It is relevant because computing skills are critical for science & society; we\'re hampered by inequity. Let\'s change this. Some references that frame this thinking:

Reviewer: ```suggestion
This talk also explicitly addressed: why is this relevant to the Exascale community? It is relevant because computing skills are critical for science & society; we're hampered by inequity. Let's change this. Some references that frame this thinking:
```

-   [Unmet needs for analyzing biological big data](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005755) - Barone et al. 2017

-   [Barriers to integration of bioinformatics into undergrad life sci education](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0224288) - Williams et al. 2019

> -   \"[Technology can exacerbate inequities if the corresponding social infrastructure is not in place](https://www.ruhabenjamin.com/viral-justice)\" - Benjamin 2022

Reviewer: ```suggestion
-   "[Technology can exacerbate inequities if the corresponding social infrastructure is not in place](https://www.ruhabenjamin.com/viral-justice)\" - Benjamin 2022
```

>
> Tooling like Jupyter Notebooks, RMarkdown, and now Quarto enables researchers to combine narrative, code, and outputs like figures in the same document, enabling transparent and reproducible workflows that can be shared openly on GitHub. This same setup can be used to teach professional researchers to code so that the teaching narrative unfolds interspersed with code the learner can run. Because the resources are openly available, and learners can work and learn with the same tools used to create those resources, there is consistency between what they see in the materials they learn from, what they see on their end, and the expected outcomes. But what is truly remarkable is the culture shift enabled by this setup; learners have in front of them these documents that reinforce what the teacher screen shares live, and which continue to be there for reference and self-paced learning when the teacher isn't there at all. This approach builds trust and resilience with learners, and role-models a new way of working openly --- all of this is a paradigm shift for how professional researchers learn new skills, work, and then teach others in turn.

> Thank you for this opportunity as a BSSw Fellow to learn more with the open source and Better Scientific Software communities! I appreciate all the work that goes into open infrastructure -- both technical and social -- as well as development and teaching. I\'m looking forward to staying connected and seeing what this community continues to do!

Reviewer: ```suggestion
Thank you for this opportunity as a BSSw Fellow to learn more with the open source and Better Scientific Software communities! I appreciate all the work that goes into open infrastructure -- both technical and social -- as well as development and teaching. I'm looking forward to staying connected and seeing what this community continues to do!
```
